dftimewolf/lib/collectors/gcloud.py

```diff
@@ -89,6 +92,24 @@ def Process(self) -> None:
           platform='gcp')
       self.state.StoreContainer(container)
 
+    if self.stop_instance and not warned:
+      if self.remote_instance_name:
+        try:
+          remote_instance = self.remote_project.compute.GetInstance(
+              self.remote_instance_name)
+          # TODO(dfjxs): Account for GKE Nodes
+          remote_instance.Stop()
+        except RuntimeError as exception:
+          self.ModuleError(str(exception), critical=False)
+        self.logger.success(
+            'Stopped instance {0:s}'.format(self.remote_instance_name))
+      else:
+        self.logger.warning(
+            'Could not stop instance, instance name not provided')
```

Comment:
Should we add this warning to the SetUp phase? Maybe even bail if the user provides that flag and no instance name?

Reply:
Ah, good point. Will do.